### PR TITLE
Update TF parsing

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
* We don't need to parse `xunit` (=the single test results) anymore.
* We can newly get `log_url` from the API without a need to hardcode it.

TODO:
* [ ] Wait for Miro (=@thrix) to add the `log_url` to their API. (edit: it will be [run.artifacts](https://testing-farm.gitlab.io/api/#operation/requestsGet).)

### CODE REVIEW

File: packit_service/worker/handlers/testing_farm_handlers.py [hunk 1]
```diff
@@ -178,14 +174,6 @@ def run(self) -> TaskResults:
             status = CommitStatus.failure
             summary = self.summary or "Tests failed ..."
 
-        if len(self.tests) > 0 and self.tests[0].name == "/packit/install-and-verify":
```
Comment (jpopelka) [inline packit_service/worker/handlers/testing_farm_handlers.py:181]: aha, so the installability test has been the use case for the xunit parsing.
I see you solved it elsewhere so we should be good to go :-)

**Review by jpopelka — APPROVED** [review]
thanks!

**Review by TomasTomecek — APPROVED** [review]
very nice, goodbye all the code :)

**Comment by lachmanfrantisek** [comment]
recheck

**Comment by lachmanfrantisek** [comment]
recheck

**Comment by lachmanfrantisek** [comment]
recheck

**Pull request merged**